src/Telemetry/AzureDevOpsTimeline/KustoTimelineTelemetryRepository.cs

```diff
@@ -0,0 +1,197 @@
+using Kusto.Data.Common;
+using Kusto.Data.Exceptions;
+using Kusto.Data.Net.Client;
+using Kusto.Ingest;
+using Microsoft.DotNet.Internal.AzureDevOps;
+using Microsoft.DotNet.Kusto;
+using Microsoft.DotNet.Services.Utility;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.AzureDevOpsTimeline
+{
+
+    public class KustoTimelineTelemetryRepository : ITimelineTelemetryRepository
+    {
+        private readonly ILogger<KustoTimelineTelemetryRepository> _logger;
+        private readonly IKustoIngestClient ingest;
```

Comment:
The naming here is inconsistent. :-) They should be _injest, _query and _disposedValue to math the others (and the naming guidlines)